packages/fxa-settings/src/pages/CompleteSignin/en.ftl

```diff
@@ -3,4 +3,4 @@
 # This is a label that precedes any error which could arise from trying to validate the user's signin
 error-label = Error:
 # This is a message that is shown to users along with a "Loading" spinner while the site tries to check their signin
-validating-signin = Validating signin...
+validating-signin = Validating sign-in…
```

Comment:
/summon @bcolsson @flodolo 
If this changes to the correct/better &hellip; do we also need to change the FTL ID to notify localizers that a new string appeared? Or Is there an alternate/preferred way (somebody hand tweaking locale files in Git directly to avoid _n_ localizers redoing a string)

Reply:
Thanks for the ping. I requested this change via slack. This was something I caught before merging the previous string, so no need to change the ID as it has not been exposed yet.

Normally, if it were an existing string we would want to update the ID.